helix-term/src/application.rs

```diff
@@ -1127,10 +1127,10 @@ impl Application {
             ..
         } = params;
 
-        let path = match uri.to_file_path() {
+        let path = match uri_to_file_path(&uri) {
             Ok(path) => path,
             Err(err) => {
-                log::error!("unsupported file URI: {}: {:?}", uri, err);
+                log::error!("{err}: {uri}");
```

Comment:
if we are going to create a custom error method just include the URI in the error straight away so we don't have to do custom error formatting everywhere